js/ui/html_editor/modules/base.js

```diff
@@ -20,6 +20,10 @@ if(Quill) {
         addCleanCallback(callback) {
             this.editorInstance.addCleanCallback(callback);
         }
+
+        handleOptionChangeValue(value) {
+            Object.entries(value).forEach(([name, value]) => this.option(name, value));
```

Comment:
```suggestion
        handleOptionChangeValue(changes) {
            Object.entries(changes).forEach(([name, value]) => this.option(name, value));
```

Do not use the same name in the different context